notebooks/paper_analysis/dc6_functions.py

```diff
@@ -147,6 +157,13 @@ def get_diagnostics_filepath(diag_type, data_type, institutions, ensemble_member
         file_var_name = variable
         var_name = variable
         
+    # load data paths 
+    # load data paths
+    with open('/home/jovyan/downscaleCMIP6/notebooks/downscaling_pipeline/post_processing_and_delivery/data_paths.yaml', 'r') as f:
```

Comment:
noting the absolute filepath. this could probably be made relative if we're being strict

Reply:
adding as a to-do for later